src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -172,60 +201,73 @@ def _make_array_shape_and_strides(
     def _add_storage(
         self,
         sdfg: dace.SDFG,
+        known_symbols: set[str],
         name: str,
-        symbol_type: ts.DataType,
+        gt_type: ts.DataType,
         transient: bool = True,
-        is_tuple_member: bool = False,
     ) -> list[tuple[str, ts.DataType]]:
         """
-        Add storage for data containers used in the SDFG. For fields, it allocates dace arrays,
-        while scalars are stored as SDFG symbols.
+        Add storage in the SDFG for a given GT4Py data symbol.
 
-        The fields used as temporary arrays, when `transient = True`, are allocated and exist
-        only within the SDFG; when `transient = False`, the fields have to be allocated outside
-        and have to be passed as array arguments to the SDFG.
+        GT4Py fields are allocated as dace arrays. GT4Py scalars are represented
+        as scalar objects in the SDFG; the exception are the symbols passed as
```

Comment:
I would write "DaCe scalar objects" to make it clear because we have to distinguish between different scalars (that are completely different) here.

Reply:
Done.